test/contrib/rack/helpers.rb

```diff
@@ -1,9 +1,12 @@
 require 'helper'
+require 'net/http'
 require 'ddtrace'
 require 'ddtrace/contrib/rack/middlewares'
 
 require 'rack/test'
 
+Datadog::Monkey.patch_module(:http)
```

Comment:
not sure here, should we automatically patch `net/http` when we require the rack helpers? can't we do this in the test case `setUp` handler? Just asking because AFAIR patching the library at import time was hiding a bug. Mostly a question rather than requiring an action.